src/main/kotlin/org/lsst/dax/albuquery/Results.kt

```diff
@@ -40,6 +40,18 @@ import java.sql.SQLException
 
 import java.util.NoSuchElementException
 
+/*
+* For keeping track of query phase (state).
+*/
+data class PhaseInfo(
+    var identifier: String = "",
+    var phase: String = "",
+    var parameters: String = "",
+    var format: String = "",
```

Comment:
Do you ever use this format parameter here?

Reply:
It's used to keep track of the user-specified format for response.

Reply:
Sorry maybe this is my lack of kotlin understanding, but where do you set phaseInfo.format, I don't see it.  I get what it's supposed to be used for, but I don't ever see it set (and you check headers and RESPONSEFORMAT in both places)

Reply:
See comment below on Async.createAsyncQuery().